kpatch-build/create-diff-object.c

```diff
@@ -612,12 +655,34 @@ static void kpatch_compare_sections(struct list_head *seclist)
 			if (sec->base->sym && sec->base->sym->status != CHANGED)
 				sec->base->sym->status = sec->status;
 		} else {
-			if (sec->sym && sec->sym->status != CHANGED)
-				sec->sym->status = sec->status;
+			struct symbol *sym = sec->sym;
+
+			if (sym && sym->status != CHANGED)
+				sym->status = sec->status;
+
+			if (sym && sym->child && sym->status == SAME &&
+					sym->child->sec->status == CHANGED)
```

Comment:
The alignment is off here, it should look like:
```
			if (sym && sym->child && sym->status == SAME &&
			    sym->child->sec->status == CHANGED)
```

Reply:
Ah, my vim's tabstop/shiftwidth combination made it look like it was aligned.